app/ember/src/client/preview/utils.js

```diff
@@ -0,0 +1,21 @@
+import { document } from 'global';
+
+/**
+ * gets the application config from meta tag
+ * @return {Object} config object
+ */
+export function getApplicationConfig() {
+  let config = {};
+  const metas = document.getElementsByTagName('meta');
```

Comment:
Instead of parsing the config directly, we should do something like:

```js
return window.require(`${APP.name}/config/environment').default;
```

The `config/environment` module itself contains the appropriate logic to parse the config (in some cases it doesn't come from a meta tag).